doc/tutorials/nires_howto.rst

```diff
@@ -0,0 +1,435 @@
+frames, we're indicating that the OH sky lines should be used to perform the
+wavelength calibration.  The wavelength calibration for these two sets of frames
+will be performed *independently* because we assign them to different
+:ref:`calibration-groups`.
+
+The observations of the standard star in this dataset are sufficiently long that
+there is good signal in the sky lines that allows for use of the OH lines for
+its wavelength calibration.  More generally, beware that short exposures may not
+allow for this, in which case you should not designate your ``standard`` frames
+as also being ``arc,tilt`` frames and you must assign the standard frames to a
+calibration group that includes ``arc,tilt`` frames.  In this example, that
+would mean assigning all 4 science and standard frames to the same calibration
+group.
+
+The main data-reduction script (:ref:`run-pypeit`) does *not* perform the
+telluric correction; this is done by a separate script.  However, and even if
+you don't intend to telluric-correct or flux-calibrate your data, it's useful to
+include the standard star observations along with the reductions of your main
+science target, particularly if the science target is faint.  If your object is
+faint, tracing the object spectrum for extraction can be difficult using only
+the signal from the source itself.  PypeIt will resort to a "tracing crutch" if
+the source signal becomes too weak.  Without the bright standard star trace, the
+tracing crutch used is the slit/order edge, which will not include the effects
+of differential atmospheric refraction on the object centroid and therefore
+yield a poorer spectral extraction.
+
+Dither sequence
+---------------
+
+As stated above, the automated algorithms in :ref:`pypeit_setup` don't correctly
+set the dither pattern for this dataset, as handled by the
+``comb_id`` and ``bkg_id`` columns.  See :ref:`here <nires_config_report>` for
+an example where the frame typing and dither pattern determinations *are*
+correct by default.
+
+In this example dataset, the science object and the standard star are both only
+observed at two offset positions.  We use the :ref:`pypeit_file` (see the
+corrected version above) to indicate this using the ``comb_id`` and ``bkg_id``
+columns, as described by :ref:`a-b_differencing` (specifically, see
+:ref:`ab-image-differencing`).
+
+By setting ``comb_id=1`` and ``bkg_id=2`` for frame ``s190519_0059.fits``, we
+are indicating that this frame should be treated as frame "1" (A) and that frame
+"2" (B) should be used as its background.  We reverse the values of ``comb_id``
+and ``bkg_id`` for frame ``s190519_0060.fits``, which indicates that this frame
+should be treated as frame "2" (B) and that frame "1" (A) should be used as its
+background.  I.e., the ``comb_id`` column effectively sets the numeric identity
+of each frame, and the ``bkg_id`` column selects the numeric identity of the
+frame that should be used as the background image.
```

Comment:
I think it is worth just doing this for ABBA which is the default dither pattern. But not required for this PR.